doc/classes/CPUParticles2D.xml

```diff
@@ -13,53 +17,74 @@
 		</member>
 		<member name="angle_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Rotation randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="angular_velocity" type="float" setter="set_param" getter="get_param">
+			Initial angular velocity applied to each particle. Sets the speed of rotation of the particle.
 		</member>
 		<member name="angular_velocity_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's angular velocity will vary along this [Curve].
 		</member>
 		<member name="angular_velocity_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Angular velocity randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="anim_offset" type="float" setter="set_param" getter="get_param">
+			Particle animation offset.
 		</member>
 		<member name="anim_offset_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's animation offset will vary along this [Curve].
 		</member>
 		<member name="anim_offset_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Animation offset randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="anim_speed" type="float" setter="set_param" getter="get_param">
+			Particle animation speed.
 		</member>
 		<member name="anim_speed_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's animation speed will vary along this [Curve].
 		</member>
 		<member name="anim_speed_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Animation speed randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="color" type="Color" setter="set_color" getter="get_color">
+			Each particle's initial color. If [member texture] is defined, it will be multiplied by this color.
 		</member>
 		<member name="color_ramp" type="Gradient" setter="set_color_ramp" getter="get_color_ramp">
+			Each particle's color will vary along this [Gradient].
 		</member>
 		<member name="damping" type="float" setter="set_param" getter="get_param">
+			The rate at which particles lose velocity.
 		</member>
 		<member name="damping_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Damping will vary along this [Curve].
 		</member>
 		<member name="damping_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Damping randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="CPUParticles2D.DrawOrder">
+			Particle draw order. Uses [enum DrawOrder] values. Default value: [constant DRAW_ORDER_INDEX].
 		</member>
 		<member name="emission_colors" type="PoolColorArray" setter="set_emission_colors" getter="get_emission_colors">
```

Comment:
Same thing here, three properties missed.

Reply:
That's mainly because I was copying existing documented properties.